### PR TITLE
Small Updates / Quick Wins #228

### PR DESCRIPTION
**Pull request by markwilliamfirth**
#228

Minor updates to site:
- Removal of Anita
- Removal of chaggosians reference
- Formatting updates
- Updating offering
- Updating mission

This is not a complete update - there are still many changes that need doing, these are just minor ones that I spotted as I was updating Anita's team status

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <div class="pure-g full-height-block dark-bg splash">
     <a href="/team"><div class="splash-container"></div></a>
     <div class="pure-u-1 splash-tagline">
-      <span>dwyl is a <span class="tagline-highlight">team</span> of people using <span class="tagline-highlight">technology</span> to solve real problems</span>
+      <span>dwyl /dwɪɫ/ is a <span class="tagline-highlight">worldwide community</span> of people using <span class="tagline-highlight">technology</span> to solve real problems.</span>
```
Comment (iteles) [inline views/index.html]: Does anyone really understand that phonetic thing?

File: views/index.html [hunk 2]
```diff
@@ -18,27 +18,27 @@
...
+              We build responsive, mobile-first MVPs with intuitive and human-centred design.
             </p>
         </div>
       </div>
       <div class="pure-u-1 pure-u-md-1-3 flex-center section-box-wrapper">
         <div class="section-box">
           <div class="offering-icon user-testing-icon">
           </div>
-          <h3>User testing</h3>
+          <h3>Co-working</h3>
```
Comment (iteles) [inline views/index.html]: I don't mind this change given it will only be here for two weeks, but the importance of user testing needs to be highlighted in our new content.
Good to get this on here though 👍

File: views/index.html [hunk 3]
```diff
@@ -194,10 +194,7 @@ <h2 class="call-to-action">Let's Talk!</h2>
   <!-- This needs an additional class of dark-bg when 'What our clients say' gets added -->
   <div class="copyright-block">
     <p>
-      Copyright 2016 &copy; dwyl ltd, company number 09525434 registered in England &amp; Wales at 104 Mary Datchelor Close, London SE5 7DY, UK. All rights reserved.
-    </p>
-    <p>
-      <i>We fully support the <a href="http://www.chagossupport.org.uk/">Chagossians</a> and their right to claim profits from the sale of .io domains associated with their homeland.</i>
+      Copyright 2016 &copy; DWYL LTD, company number 09525434 registered in England &amp; Wales at 104 Mary Datchelor Close, London SE5 7DY, United Kingdom. All rights reserved.
```
Comment (iteles) [inline views/index.html]: Wanna change this to 2017? 😉

File: views/index.html [hunk 4]
```diff
@@ -18,27 +18,27 @@
...
       </div>
       <div class="pure-u-1 pure-u-md-1-3 flex-center section-box-wrapper">
         <div class="section-box">
           <div class="offering-icon training-icon">
           </div>
-          <h3>Training</h3>
+          <h3>Technical Training</h3>
             <p>
-              We provide professional training for developers in TDD, Node.js and React.
+              Hands-on, immersive learning designed to enlighten, inspire and empower. Workshops in agile development, product ownership, human-centered design and more.
```
Comment (iteles) [inline views/index.html]: Did you run the code to make sure that having a way bigger caption for this picture than the others won't make it look weird or break the styling?

File: views/index.html (hunk 1 above)
Comment (markwilliamfirth) [inline views/index.html]: I do...not sure if other people do...this comes from https://github.com/dwyl/video/issues/24 and https://github.com/dwyl/hq/issues/249 - feel free to remove if we don't want it on there!

File: views/index.html (hunk 2 above)
Comment (markwilliamfirth) [inline views/index.html]: I thought it was against our values to use the word "user" 😆  mentioned "human-centred design" in it's place in training bit

File: views/index.html (hunk 4 above)
Comment (markwilliamfirth) [inline views/index.html]: Didn't notice that - does it do that?

File: views/index.html (hunk 3 above)
Comment (markwilliamfirth) [inline views/index.html]: Can do next time! As I said there are lots more changes to make 😛

File: views/index.html (hunk 1 above)
Comment (Jbarget) [inline views/index.html]: i dont

File: views/index.html (hunk 2 above)
Comment (iteles) [inline views/index.html]: 😊 User testing is a well-known term.

File: views/index.html (hunk 4 above)
Comment (iteles) [inline views/index.html]: Took me 2 mins to pull this branch, `npm i && npm start`. Always worth running the code you're changing.

screen shot 2017-06-01 at 16 34 08

File: views/index.html (hunk 2 above)
Comment (markwilliamfirth) [inline views/index.html]: @iteles I know it is - I just thought @nelsonic didn't like using the word user - I remember him mentioning it in a meeting

File: views/index.html (hunk 4 above)
Comment (markwilliamfirth) [inline views/index.html]: Do you want them all the same size?

Comment (iteles) [inline views/index.html]: No... I just want it to not look ugly ;)

Comment (markwilliamfirth) [inline views/index.html]: Do I have to push another commit to this PR? Not sure how to do this...never done this before...

File: views/index.html (hunk 1 above)
Comment (iteles) [inline views/index.html]: @markwilliamfirth I'll leave it to you to decide whether this looks confusing or not:
screen shot 2017-06-01 at 16 40 25

File: views/index.html (hunk 2 above)
Comment (iteles) [inline views/index.html]: Think SEO on this one. We just don't refer to our _community_ as users: https://github.com/dwyl/start-here/blob/master/manifesto.md#we-do-not-have-users-we-have-people

File: views/index.html (hunk 4 above)
Comment (iteles) [inline views/index.html]: You went through Founders & Coders, of course you have! 😆 
Yea, just push another commit to this branch and it will show up in this PR.

**Comment by markwilliamfirth** [comment]
:shipit:
